docs/api/alerts/find.asciidoc

```diff
@@ -0,0 +1,120 @@
+[[alerts-api-find]]
+=== Find alerts API
+++++
+<titleabbrev>Find alerts</titleabbrev>
+++++
+
+Retrieve a paginated set of alerts based on condition.
+
+[[alerts-api-find-request]]
+==== Request
+
+`GET <kibana host>:<port>/api/alerts/_find`
+
+[[alerts-api-find-query-params]]
+==== Query Parameters
+
+`per_page`::
+  (Optional, number) The number of alerts to return per page.
+
+`page`::
+  (Optional, number) The page number.
+
+`search`::
+  (Optional, string) An Elasticsearch {ref}/query-dsl-simple-query-string-query.html[simple_query_string] query that filters the alerts in the response.
+
+`default_search_operator`::
+  (Optional, string) The operator to use for the `simple_query_string`. The default is 'OR'.
+
+`search_fields`::
+  (Optional, array|string) The fields to perform the `simple_query_string` parsed query against.
+
+`fields`::
+  (Optional, array|string) The fields to return in the `attributes` key of the response.
+
+`sort_field`::
+  (Optional, string) Sorts the response. Includes "root" and "type" fields. "root" fields exist for all saved objects, such as "updated_at".
```

Comment:
Since this is the alert API, do we need this explanation about root and type fields? Shouldn't this only search over saved objects of type `alert`?